src/collector.js

```diff
@@ -76,6 +77,18 @@ export default class Collector {
     return false;
   }
 
+  recordScannedFile(filename, scanner) {
+    // TODO: Add some code that verifies and normalizes `filename`
```

Comment:
Can you add an issue for this and add it to the code? That way we don't forget 😄